packages/@okta/vuepress-site/docs/guides/customize-tokens-groups-claim/add-groups-claim-org-as/index.md

```diff
@@ -15,10 +15,14 @@ Use these steps to create a Groups claim for an OpenID Connect client applicatio
 
 To test the full authentication flow that returns an ID token, build your request URL. The scopes that you need to include as query parameters are `openid` and `groups`. For the specific steps on building the request URL, receiving the response, and decoding the JWT, see <GuideLink link="../request-token-claim">Request a token that contains the custom claim</GuideLink>.
 
-> **Note:** In this example, the user signing in to your app is assigned to a group called "IT" as well as being a part of the "Everyone" group.
+> **Note:** The examples in this guide use the [Implicit flow](/docs/concepts/oauth-openid/#implicit-flow) flow for quick testing. In the following example we are requesting only `id_token` as the `response_type` value [fat ID token](/docs/concepts/api-access-management/#tokens-and-scopes), which means that the ID token is the only token included in the response. The ID token contains any groups assigned to the user that signs in when you include the `groups` scope in the request.
+
+If you want a Groups claim present in the ID token when you request both an ID token and access token, use a <GuideLink link="../add-groups-claim-custom-as">Custom Authorization Server</GuideLink>.
```

Comment:
This is about org as, you might just say if you request both id token and access token, the group claim will not be included in the id token, you can use the access token to get the group claim from the userinfo endpoint.